net/adoptopenjdk/bumblebench/string/StringValueOfBench.java

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+package net.adoptopenjdk.bumblebench.string;
```

Comment:
should probably change to adoptium naming for packaging

Reply:
We did not change the package names when migrating to Eclipse Foundation, as this was deemed unnecessary, so all other classes in this repository are in net.adoptopenjdk.bumblebench.  Not sure we need to change this, or if PMC felt we should, perhaps its a separate undertaking.

Reply:
Fair, we can do a rename / refactor across the whole repo if required.